### PR TITLE
Jeremy/eng 998 verification

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
Reimplement the account/verification using Reactstrap

Hard to test thoroughly 'cause Google recaptcha locks me out 😢 

## How to Test
- [ ] Visit work/account/verification
- [ ] Ensure user can successfully verify account via phone number
- [ ] Ensure user can successfully verify account via email
- [ ] Verify '!' disappears after both are verified

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/pages/Account/AccountVerification.tsx [hunk 1]
```diff
@@ -1,11 +1,173 @@
...
+          color: 'success',
+          msg: getDisplaySuccessMessage(SuccessMessage.EMAIL_VERIFICATION_SENT),
+          show: true,
+        })
+      })
+      .catch((error: any) => {
+        console.error(error);
+        setAlert({
+          color: 'success',
+          msg: error,
```
Comment (woeltjen) [inline src/pages/Account/AccountVerification.tsx]: Should this be `error.message`? Not sure what the failure cases are for `sendEmailVerification`

Comment (jerminatorhits) [reply src/pages/Account/AccountVerification.tsx]: ah thanks!

**Comment by kcvan** [comment]
@jerminatorhits if you add your # to the test numbers on firebase it wont lock you out

**Comment by jerminatorhits** [comment]
@kcvan o snap that's cool. thx!

**Pull request merged**